programs/bpf_loader/src/lib.rs

```diff
@@ -3833,15 +3868,14 @@ mod tests {
         let mut file = File::open("test_elfs/out/sbpfv3_return_ok.so").expect("file open failed");
         let mut elf = Vec::new();
         file.read_to_end(&mut elf).unwrap();
-        deploy_program!(
+        deploy_program(
```

Comment:
Why this change here specifically?

Reply:
It was to add some test coverage for the new function. I'll likely revert it to use the macro, since we are renaming `deploy_program_internal()` for the public API.